Hymns/Tags/TagSheetView.swift

```diff
@@ -42,21 +42,23 @@ struct TagSheetView: View {
                             }, label: {
                                 HStack {
                                     Text(tag.title).font(.body).fontWeight(.bold)
-                                    Image(systemName: "checkmark.circle.fill")
+                                    Image(systemName: "xmark.circle.fill")
                                 }
                                 .tagPill(backgroundColor: tag.color.background, foregroundColor: tag.color.foreground)
                             }).padding(2)
                         }
                     }
+                    if self.viewModel.tagsForHymn.isEmpty && !self.viewModel.otherTags.isEmpty {
+                        Text("Tags").font(.headline).fontWeight(.bold)
+                        Text("\n ")
```

Comment:
Imo it looks much better if tags is still there empty when there are no tags instead of disappearing. Since no matter what the user has come to this page to do something with tags. This gives a more consistent feel to the page since "Tags" is always at the top of the sheet instead of appearing and disappearing when tags are added or removed.